home-manager/systemd.nix

```diff
@@ -0,0 +1,47 @@
+{ pkgs, lib, ... }:
+
+{
+  # https://github.com/nix-community/home-manager/blob/release-24.05/modules/systemd.nix#L161-L173
+  systemd = {
+    user = {
+      services.podman = {
```

Comment:
Referred https://github.com/simonoscr/dotfiles/blob/81669fac813dd518e06895793a141abad9967dc8/modules/home-manager/podman/podman.nix Thank you! 🙏